release/release-offline.ts

```diff
@@ -265,7 +265,8 @@ async function s3() {
   );
 
   if (isLatest === 'true') {
-    await $`aws s3 cp ${JAR_PATH}/metabase.jar s3://${AWS_S3_DOWNLOADS_BUCKET}/latest/metabase.jar`.pipe(
+    const latestPath = edition === "ee" ? `enterprise/latest` : `latest`;
```

Comment:
when I went to use the offline script to fix it I discovered that it pushed both ee and oss to `/latest`, when ee should go to `enterprise/latest`